libnavigation-core/src/main/java/com/mapbox/navigation/core/test/TripServiceActivity.kt

```diff
@@ -0,0 +1,115 @@
+package com.mapbox.navigation.core.test
+
+import android.os.Bundle
+import android.os.SystemClock
+import android.widget.Button
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.annotation.module.MapboxModuleType
+import com.mapbox.base.common.logger.Logger
+import com.mapbox.base.common.logger.model.Message
+import com.mapbox.base.common.logger.model.Tag
+import com.mapbox.common.module.provider.MapboxModuleProvider
+import com.mapbox.navigation.base.TimeFormat.TWENTY_FOUR_HOURS
+import com.mapbox.navigation.base.internal.VoiceUnit.METRIC
+import com.mapbox.navigation.base.internal.extensions.inferDeviceLocale
+import com.mapbox.navigation.base.options.NavigationOptions
+import com.mapbox.navigation.base.trip.notification.TripNotification
+import com.mapbox.navigation.core.R
+import com.mapbox.navigation.core.Rounding
+import com.mapbox.navigation.core.internal.MapboxDistanceFormatter
+import com.mapbox.navigation.core.trip.service.MapboxTripService
+import com.mapbox.navigation.utils.internal.ThreadController
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.isActive
+import kotlinx.coroutines.launch
+
+internal class TripServiceActivity : AppCompatActivity() {
+
+    private var mainJobController = ThreadController.getMainScopeAndRootJob()
+    private lateinit var tripNotification: TripNotification
+    private lateinit var mapboxTripService: MapboxTripService
+    private var textUpdateJob: Job = Job()
+    private val dummyLogger = object : Logger {
+        override fun d(tag: Tag?, msg: Message, tr: Throwable?) {}
+
+        override fun e(tag: Tag?, msg: Message, tr: Throwable?) {}
+
+        override fun i(tag: Tag?, msg: Message, tr: Throwable?) {}
+
+        override fun v(tag: Tag?, msg: Message, tr: Throwable?) {}
+
+        override fun w(tag: Tag?, msg: Message, tr: Throwable?) {}
+    }
+    private lateinit var toggleNotification: Button
+    private lateinit var notifyTextView: TextView
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_trip_service)
+
+        toggleNotification = findViewById(R.id.toggleNotification)
+        notifyTextView = findViewById(R.id.notifyTextView)
```

Comment:
kotlin android extensions to avoid `findViewById` 😃

Reply:
have tried, didn't succeed. Maybe because it's a `lib` module? Do I need any extra dependency? (don't remember )))

Reply:
ah, need to add `apply plugin: 'kotlin-android-extensions'`
maybe keep `findViewById ` to avoid additional dependencies in core module? We are not going to use `findViewById` often

Reply:
I'm ok with both variants, because plugin doesn't bring a huge value, so up to you

Reply:
let's keep it as is, we can change it, if we will add more activities to `core` module